web/templates/character/_show_talents.html.eex

```diff
@@ -2,12 +2,16 @@
 <div class="row">
   <table class="table table-condensed">
     <tr>
+      <th class="col-md-1">Rank</th>
```

Comment:
#nitpick-mode In every other case on the site, the first column has been the most important field. Generally this has been the name of a skill / weapon / force power. As such, I think Name makes more sense then as the first field for Talents, but if you disagree then I'm happy to merge as-is

Reply:
You bet - updated.